addons/l10n_eu_oss/tests/test_oss.py

```diff
@@ -9,7 +9,7 @@
 class OssTemplateTestCase(AccountTestInvoicingCommon):
 
     @classmethod
-    def load_specific_chart_template(cls, chart_template_ref):
+    def setUpClass(cls, chart_template_ref):
```

Comment:
Why was it not like this in the first place?

Reply:
I don't remember, I think the reason is just "I'm dumb".
I fixed that fastly like a moron for a l10n nightly build and I got r+ed at the same speed.
I hadn't the opportunity to look it back and later on when I realized I hadn't a good reason to touch that file.